tests/test_url.py

```diff
@@ -62,6 +62,17 @@ def test_safe_url_string(self):
     def test_safe_url_string_unsafe_chars(self):
         safeurl = safe_url_string(r"http://localhost:8001/unwise{,},|,\,^,[,],`?|=[]&[]=|")
         self.assertEqual(safeurl, r"http://localhost:8001/unwise%7B,%7D,|,%5C,%5E,[,],%60?|=[]&[]=|")
+        
+    def test_safe_url_string_quote_path(self):
+        safeurl = safe_url_string(u'http://google.com/"hello"', self.encoding, True)
```

Comment:
I think it would be better to use a keyword argument here for both encoding and path quoting, as done in the other tests, to make the test more readable.